lib/utils/queries.dart

```diff
@@ -68,10 +68,10 @@ class Queries {
   }
 
   //login the user
-  String loginUser(String email, String password) {
+  String loginUser(String email, String password, String? token) {
```

Comment:
What is this new token parameter being used for? I noticed an empty string being passed when referenced

Reply:
Removed the token string.